.pre-commit-config.yaml

```diff
@@ -101,7 +101,7 @@ repos:
     hooks:
       - id: codespell
         entry: codespell -I codespell.txt ./doc/source
-        exclude: ^assets/.*$|^charts/.*$|^docs/assets/.*$|^eclipse-cheatsheets-to-dita-to-pdf/.*$|^Gemfile\.lock$|^go\.sum$|^go\.work\.sum$|^logorama/.*$|^rubycritic/.*$|^ruby-eclipse-cheatsheets-to-dita/.*$|^tests/spelling/.*$|^seek/job_ind\.json$
+        exclude: ^assets/.*$|^package-lock.json$|^charts/.*$|^docs/assets/.*$|^eclipse-cheatsheets-to-dita-to-pdf/.*$|^Gemfile\.lock$|^go\.sum$|^go\.work\.sum$|^logorama/.*$|^rubycritic/.*$|^ruby-eclipse-cheatsheets-to-dita/.*$|^tests/spelling/.*$|^seek/job_ind\.json$
```

Comment:
```suggestion
        exclude: ^assets/.*$|^package-lock\.json$|^charts/.*$|^docs/assets/.*$|^eclipse-cheatsheets-to-dita-to-pdf/.*$|^Gemfile\.lock$|^go\.sum$|^go\.work\.sum$|^logorama/.*$|^rubycritic/.*$|^ruby-eclipse-cheatsheets-to-dita/.*$|^tests/spelling/.*$|^seek/job_ind\.json$
```

Reply:
thanks for your review. looks good - thanks for fixing that mistake for `.`.